docs/cFE Application Developers Guide.md

```diff
@@ -179,10 +179,10 @@ software interface.
 
 ## 1.3 Applicable Documents
 
-| **Location**                                 | **Documents**         | **Description**    |
-|:---------------------------------------------|:----------------------|:-------------------|
-| https://github.com/nasa/osal/tree/master/doc |	OSAL Library API.doc | Describes OSAL API |
-| TBD                                          |	PSP API (TBD)        | Describes PSP API  |
+| **Location**                                  | **Documents**         | **Description**    |
+|:----------------------------------------------|:----------------------|:-------------------|
+| https://github.com/nasa/osal/tree/main/docs   | OSAL Library API.doc  | Describes OSAL API |
+| TBD                                           | PSP API (TBD)         | Describes PSP API  |
```

Comment:
It was broken before and it's broken still... I plan to fix so could just leave out.  Really should reference wherever the deployed document ends up (or just remove direct references).

Reply:
Yeah, the link doesn't work this instant, but as soon as this was merged to "main" it should start working.  Nonetheless, I can remove/undo this change if you are separately fixing that one, no problem.

Reply:
Fixed in 888d292b

Reply:
The API is no longer a *.doc, it's generated and deployed from doxygen, and it goes in a separate branch.  Right now it's in the cFS repo.